vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs

```diff
@@ -54,6 +55,28 @@ type internal FSharpCompletionProvider
                 | ClassificationTypeNames.NumericLiteral -> false
                 | _ -> true // anything else is a valid classification type
             ))
+
+    static let mruItems = Dictionary<(* Item.FullName *) string, (* hints *) int>()
+
+    /// Normalizes hints to monothonically increasing sequence ("name1" => 1, "name2" => 110, "name3" => 25) to ("name1" => 1, "name2" => 3, "name3" => 2)
+    static let getNormalizedMruHints () =
+        let items = mruItems |> Seq.map (fun (KeyValue(fullName, hints)) -> fullName, hints) |> Seq.sortBy snd |> Seq.toList
+        match items with
+        | [] -> mruItems
+        | _ ->
+            items
```

Comment:
Please use `||>`` for fold operations, i.e. ``(state,bs) ||> Seq.fold (fun state b -> ...)``